05_syntax_lab/02.py

```diff
@@ -1,6 +1,8 @@
-"""
-Write a Python program that randomizes 7 numbers
-and prints their sum total.
-If the sum is divisable by 7, also print the word "Boom"
-"""
-
+# 7 boom
+for i in range (1,8):
+    import random
+    x = random.randint (1,101)
+    sum =+ x
+print "the sum of the random numbers is:", sum 
+if sum % 7 == 0:
+    print "boom"

```

Comment:
👍 
The ```import random``` should always be at the beginning of the program. Outside of any conditions or loops.